src/clients/storage/StorageClientBase-inl.h

```diff
@@ -200,12 +200,13 @@ folly::Future<StatusOr<Response>> StorageClientBase<ClientType, ClientManagerTyp
         auto ex = exWrapper.get_exception<TransportException>();
         if (ex && ex->getType() == TransportException::TIMED_OUT) {
```

Comment:
maybe better to handle the error as the following way:

```cpp
if (ex) {
  // Transport exception
  if (ex->getType() == TIMEOUT) {
    // report time out error
  } else {
    // report transport error
  }
} else {
  // report other exception
}
```

Reply:
Fixed.